torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -734,6 +736,41 @@ llvm::Error PyTorchModelLoader::loadMin(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], output);
 }
 
+llvm::Error PyTorchModelLoader::loadFlatten(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, 1));
+
+  // Indexes of aten::flatten inputs.
+  struct Inputs {
+    enum {
+      input = 0, // NCHW
+      start_dim = 1,
+      end_dim = 2,
+    };
+  };
+
+  glow::NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getGlowNodeValue(inputs[Inputs::input]));
+
+  auto startHandle = glow::Handle<int32_t>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      startHandle, getGlowConstantHandle<int32_t>(inputs[Inputs::start_dim]));
+  uint32_t startDim;
+  RETURN_IF_ERR(contractParamIfNeeded(startHandle, startDim));
+
+  auto endHandle = glow::Handle<int32_t>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      endHandle, getGlowConstantHandle<int32_t>(inputs[Inputs::end_dim]));
+  int32_t endDim;
+  RETURN_IF_ERR(contractParamIfNeeded(endHandle, endDim));
+  RETURN_ERR_IF_NOT(endDim == -1, "only -1 value for end_dim got supported.");
```

Comment:
it would be pretty trivial to support this as well, I think we should just do it, up to you though

Reply:
It's not clear for me now, how to do it right, leaving as TODO.